packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -154,20 +153,18 @@ class BottomSheetRangeCell extends Component {
 		);
 
 		const defaultSliderStyle = getStylesFromColorScheme( styles.sliderTextInput, styles.sliderDarkTextInput );
-		const resetButtonText = Platform.OS === 'ios' ? __( 'Reset' ) : __( 'RESET' );
-		const resetButton = { title: resetButtonText, handler: this.handleReset };
 
 		return (
 			<Cell
 				{ ...cellProps }
 				cellContainerStyle={ styles.cellContainerStyles }
 				cellRowContainerStyle={ styles.cellRowStyles }
 				accessibilityRole={ 'none' }
+				value={ '' }
```

Comment:
Just dropping a question: why do we need to add this line?

Reply:
To have a proper style for a label - without it label will be `centered`

Reply:
We do have the `leftAlign` prop. It adds a little margin to the left but I think we can remove it because from the designs looks like it shouldn't have any when it doesn't have an icon and value.

Here's the style I mentioned -> https://github.com/WordPress/gutenberg/blob/dc229b4f937761dd818b46078049ae863d6d718b/packages/components/src/mobile/bottom-sheet/styles.native.scss#L137